src/hooks/custom-hooks/useExtendedOrderDetails.ts

```diff
@@ -95,60 +95,82 @@ const useExtendedOrderDetails = ({
                     : localize('Your payment details'),
                 resultString: this.isBuyOrder
                     ? localize("You've received {{currency}} {{displayAmount}}", {
-                          currency: this.account_currency,
-                          displayAmount: this.amount_display,
+                          currency: this.p2p_order_info ? this.p2p_order_info.account_currency : this.account_currency,
```

Comment:
does this.account_currency exist? why not change, the data inside the useOrderInfo to data.p2p_order_info ?

Reply:
ive just realised something, i think i can fix this issue in a better way hold on